en_us/quests/hello_bolt.md

```diff
@@ -0,0 +1,139 @@
+{% include '/version.md' %}
+
+# Hello Bolt
+
+## Quest objectives
+
+- Install Puppet Bolt
+- Verify the Puppet Bolt installation
+- Run simple commands using Bolt
+
+## Get started
+
+In this quest, you will install Puppet Bolt and run some introductory commands.
+As described in the previous quest, Bolt is a task runner that enables you
+to invokes commands on one or more target nodes. It is well-suited to
+performing actions in an ad hoc manner instead of managing a node on a
+long-term basis.
+
+Ready to get started? Run the following command on your Learning VM to begin
+this quest:
+
+    quest begin hello_bolt
+
+## Install Puppet Bolt
+
+<div class = "lvm-task-number"><p>Task 1:</p></div>
+
+Packaged versions of Bolt are available for Red Hat Enterprise Linux 6 and 7,
```

Comment:
Might be this was written a bit ago, but we also support Debian 8 and 9 and Ubuntu 14.04, 16.04 and 18.04., and (as of yesterday) Fedora 28 and 29 and OSX 14! https://puppet.com/docs/bolt/1.x/bolt_installing.html#task-4877

Reply:
Cool, I will update that

Reply:
Added more to the list of supported platforms and removed explicit version numbers to make it easier to maintain in the future.